install.sh

```diff
@@ -360,6 +368,18 @@ os_is_deb ()
     ( which apt-get && which dpkg-query ) 2>/dev/null 1>&2
 }
 
+is_python3 ()
+{
+    if [[ ${python} == 'python3' ]]; then
+        return 0
+    fi
+    if eval "${python} -c 'import sys; sys.exit(0) if sys.version_info >= (3,0) else sys.exit(1)'"; then
```

Comment:
why `eval` here?
Could just use the exit code in the `if`:
```suggestion
    if ${python} -c 'import sys; sys.exit(0) if sys.version_info >= (3,0) else sys.exit(1)'; then
```

Reply:
I recall I tried that and it didn't work. Seems you can't execute shell variable as commands without eval.